okhttp/src/main/java/okhttp3/internal/http/HttpDate.kt

```diff
@@ -65,55 +63,51 @@
+  /** Returns the date for [value]. Returns null if the value couldn't be parsed.  */
+  @JvmStatic
+  fun parse(value: String): Date? {
+    if (value.isEmpty()) {
+      return null
     }
 
-    ParsePosition position = new ParsePosition(0);
-    Date result = STANDARD_DATE_FORMAT.get().parse(value, position);
-    if (position.getIndex() == value.length()) {
+    val position = ParsePosition(0)
+    var result = STANDARD_DATE_FORMAT.get().parse(value, position)
+    if (position.index == value.length) {
       // STANDARD_DATE_FORMAT must match exactly; all text must be consumed, e.g. no ignored
       // non-standard trailing "+01:00". Those cases are covered below.
-      return result;
+      return result
     }
-    synchronized (BROWSER_COMPATIBLE_DATE_FORMAT_STRINGS) {
-      for (int i = 0, count = BROWSER_COMPATIBLE_DATE_FORMAT_STRINGS.length; i < count; i++) {
-        DateFormat format = BROWSER_COMPATIBLE_DATE_FORMATS[i];
-        if (format == null) {
-          format = new SimpleDateFormat(BROWSER_COMPATIBLE_DATE_FORMAT_STRINGS[i], Locale.US);
-          // Set the timezone to use when interpreting formats that don't have a timezone. GMT is
-          // specified by RFC 7231.
-          format.setTimeZone(UTC);
-          BROWSER_COMPATIBLE_DATE_FORMATS[i] = format;
-        }
-        position.setIndex(0);
-        result = format.parse(value, position);
-        if (position.getIndex() != 0) {
+    synchronized(BROWSER_COMPATIBLE_DATE_FORMAT_STRINGS) {
+      for (i in 0 until BROWSER_COMPATIBLE_DATE_FORMAT_STRINGS.size) {
+        val format: DateFormat = BROWSER_COMPATIBLE_DATE_FORMATS[i]
+            ?: SimpleDateFormat(BROWSER_COMPATIBLE_DATE_FORMAT_STRINGS[i], Locale.US).apply {
+              // Set the timezone to use when interpreting formats that don't have a timezone. GMT is
+              // specified by RFC 7231.
+              timeZone = UTC
+            }
+        BROWSER_COMPATIBLE_DATE_FORMATS[i] = format
+        position.index = 0
+        result = format.parse(value, position)
+        if (position.index != 0) {
           // Something was parsed. It's possible the entire string was not consumed but we ignore
           // that. If any of the BROWSER_COMPATIBLE_DATE_FORMAT_STRINGS ended in "'GMT'" we'd have
           // to also check that position.getIndex() == value.length() otherwise parsing might have
           // terminated early, ignoring things like "+01:00". Leaving this as != 0 means that any
           // trailing junk is ignored.
-          return result;
+          return result
         }
       }
     }
-    return null;
-  }
-
-  /** Returns the string for {@code value}. */
-  public static String format(Date value) {
-    return STANDARD_DATE_FORMAT.get().format(value);
+    return null
   }
 
-  private HttpDate() {
+  /** Returns the string for [value].  */
+  @JvmStatic
+  fun format(value: Date): String {
+    return STANDARD_DATE_FORMAT.get().format(value)
   }
-}
+}
```

Comment:
No newline at end of file.

Reply:
You can disregard this; our style guide doesn’t care.